include/swift/IDE/RefactoringKinds.def

```diff
@@ -50,6 +50,8 @@ CURSOR_REFACTORING(ConvertToDoCatch, "Convert To Do/Catch", convert.do.catch)
 
 CURSOR_REFACTORING(TrailingClosure, "Convert To Trailing Closure", trailingclosure)
 
+CURSOR_REFACTORING(MemberWiseInitLocalRefactoring, "Generate Member Wise Initializer", member.wise.init.local.refactoring)
```

Comment:
I'd use either `Memberwise` or `Member-wise`. There might even be a better word than that.